ConsoleApp/SnakeGame.cs

```diff
@@ -0,0 +1,616 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp
+{
+    enum Move
+    {
+        UP = 0, 
+        DOWN = 1, 
+        LEFT = 2, 
+        RIGHT = 3,
+        INITIAL = 4
+    }
+    class SnakeGame
+    {
+        public class Point
```

Comment:
if you're describing class Point - it can have only X and Y position because it is what describes Point, position is something extra in scope of this class